indexing-service/src/test/java/org/apache/druid/indexing/common/actions/SegmentTransactionalInsertActionTest.java

```diff
@@ -169,17 +164,15 @@ public void testFailTransactionalUpdateDataSourceMetadata() throws Exception
   public void testFailBadVersion() throws Exception
   {
     final Task task = NoopTask.create();
-    final SegmentTransactionalInsertAction action = SegmentTransactionalInsertAction.overwriteAction(
-        null,
-        ImmutableSet.of(SEGMENT3),
-        null
-    );
+    final SegmentTransactionalInsertAction action = SegmentTransactionalInsertAction
+        .overwriteAction(null, ImmutableSet.of(SEGMENT3), null);
```

Comment:
Is there a style guide for this in Druid?
If not, I think that formatting can be subjective in the sense that a few people may prefer the previous code. (Not a strong opinion)

Reply:
Fair point. I don't recall why I had made this change. Let me revert this with any other change that is required.

Reply:
By the way, in terms of style guide, this is what I try to follow:
https://github.com/apache/druid/blob/f1043d20bc10d3fd44a33ad89dac556d30095a20/codestyle/checkstyle.xml#L252-L296

I also wrote something up here
https://github.com/apache/druid/pull/16563#discussion_r1633131036